frontend/README.md

```diff
@@ -17,7 +19,7 @@ You will need the following things properly installed on your computer.
 
 To install the app, run the following commands in a command line:
 
-* `git clone https://github.com/fossasia/badgeyay this repository
+* `git clone https://github.com/fossasia/badgeyay` this repository
```

Comment:
`this repository` can  be removed

Reply:
Thats in installation step... why to delete this line ?

Reply:
Since it's unneeded as the code is present. Or you can write: clone this repository. Then the code within ``

Reply:
@vaibhavsingh97 @abishekvashok I don't know why, but I misunderstood your request completely in the first place. Now its fixed 👍